packages/mutation-testing-elements/src/components/mutation-test-report-app/mutation-test-report-app.component.ts

```diff
@@ -74,6 +85,11 @@ export class MutationTestReportAppComponent extends LitElement {
         this.theme = 'light';
       }
     }
+
+    // Set the default view to "mutant" when no route is selected
+    if (this.path.length === 0 || (this.path[0] !== View.mutant && this.path[0] !== View.test)) {
+      window.location.hash = `#${View.mutant}`;
```

Comment:
I noticed that this replaces the browser history, which means you need to press back twice to go to the previous page whenever the page loads. A better alternative could be:

```suggestion
      window.location.replace(toAbsoluteUrl(`${View.mutant}`));
```